### PR TITLE
Titian mosaic freezermanagement 8.6

### PR DESCRIPTION
**Pull request by KrisDTitian**
Update of AMI's to latest version of Titian Mosaic FreezerManagement 8.6.
Updated language used in template to be uniform.
Updated RDS instance versions used from t2 to t3.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: templates/titian-mosaic.template [hunk 1]
```diff
@@ -29,7 +29,7 @@ Metadata:
       - pEC2KeyPair
       - pAppServerInstanceType
     - Label:
-        default: Amazon RDS configuration
+        default: Amazon RDS Configuration
```
Comment (sshvans) [inline templates/titian-mosaic.template:32]: Can you please revert the capitalization of every word? It doesn't follow AWS writing guidelines.

File: templates/titian-mosaic.template [hunk 2]
```diff
@@ -42,57 +42,57 @@ Metadata:
...
-        default: VPC for deployment
+        default: VPC for Deployment
       pWebAccessCIDR:
-        default: Web access CIDR 
+        default: Web Access CIDR (No SG)
        
 Parameters:
   pDBTimezone:
-    Description: The time zone to set the Titian Mosaic FreezerManagement database to.
+    Description: What time zone should we set the database to?
```
Comment (sshvans) [inline templates/titian-mosaic.template:95]: It shouldn't be a question. It's a description. Please revert it back to earlier description.

File: templates/titian-mosaic.template [hunk 3]
```diff
@@ -1,15 +1,15 @@
 ---
```
Comment (sshvans) [inline templates/titian-mosaic.template:1]: Please revert the changes for descriptions. In most cases, the change in description has been made to make it shorter. The original description has been reviewed by our Tech writers, following the AWS writing guidelines.

**Review by sshvans — CHANGES_REQUESTED** [review]
Please see my review comments inline.

File: templates/titian-mosaic.template (hunk 1 above)
Comment (KrisDTitian) [reply templates/titian-mosaic.template:32]: When I look at the main branch/last commit, these appear to be capitalized in commit [b906502].
I will make my way back through these and revert for this pull.

File: templates/titian-mosaic.template (hunk 2 above)
Comment (KrisDTitian) [reply templates/titian-mosaic.template:95]: Corrected as requested.

File: templates/titian-mosaic.template (hunk 3 above)
Comment (KrisDTitian) [reply templates/titian-mosaic.template:1]: Corrected as requested.

**Comment by KrisDTitian** [comment]
Hello Shivansh,

Thank you for your comments, I have actioned the below and pushed in a new commit to remedy these issues.

Please would you be able to take another look?

Thank you,
Kind Regards,
Kris

**Comment by sshvans** [comment]
@KrisDTitian Can you address the linting errors?
Also, seems like there is something wierd going on with descriptions and parameter labels. Now it shows that it has been changed to all lowercase. If you ok, I'll revert those changes in the PR and only keep the AMI/version updates which I believe you are intending to do in this PR. Is that correct?

**Comment by KrisDTitian** [comment]
@sshvans Hello Shivansh,

Yes I am okay with that, we are looking to get the AMI/RDS version updates pushed in and one additional change at the bottom of the template which was an incorrect reference to 'ChemAxon Compound Registry' which should be 'Titian Mosaic'.

Thank you,
Kind regards,
Kris

**Comment by KrisDTitian** [comment]
@sshvans Hello Shivansh,

Just checking if everything is okay to merge now?

Thank you,
Kind regards,
Kris

**Review by sshvans — APPROVED** [review]

**Comment by sshvans** [comment]
@KrisDTitian I don't see a taskcat.yaml file in the repo for testing. It seems that we have never had that file in the repo.
For changes to move to production S3 bucket, we need to test the templates through our CI and for that it requires .taskcat.yml file in the repo. You can read more about Quick Start testing in our [docs](https://aws-quickstart.github.io/testing.html).
You can also see sample taskcat file in other repos in [Quick Start github org](https://github.com/aws-quickstart).
